crates/model/src/solver_competition.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+// These types are going to be updated once the openapi documentation gets
+// merged. Consider them a placeholder.
+
+pub type SolverCompetitionResponse = HashMap<String, SolverSettlement>;
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
+pub struct SolverSettlement {
+    objective_value: f64,
```

Comment:
If we use a `Vec` instead of a `HashMap` we should include `solver_name: String` here.